src/checkers/lemmeknow.rs

```diff
@@ -0,0 +1,7 @@
+use lemmeknow::Identify;
+
+pub fn check_lemmeknow(input: &str) -> bool {
+    // Uses lemmeknow to check if any regexes match
+    let identifier = Identify::default();
```

Comment:
Would it be faster to move this out of the function so we don't initialise it everytime?

Reply:
Yes! will do it.